controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -381,23 +381,30 @@ columns:
     default: false
     width: 10
   - id: FlowDirection
-    name: Direction
-    tooltip: The direction of the flow observed at the Node observation point.
+    name: Node Direction
+    tooltip: The interpreted direction of the flow observed at the Node observation point.
     field: FlowDirection
-    filter: direction
+    filter: flowDirection
```

Comment:
shouldn't we call the filter "nodeDirection" ? And btw, other filters are in snake case so I guess it's flow_direction or node_direction

Reply:
Sure, done in https://github.com/netobserv/network-observability-operator/pull/578/commits/553bfe230e596a0b3282d7b709032bcadc295d0f